libnmstate/nm/connection.py

```diff
@@ -105,7 +105,10 @@ def delete(self):
             if not self.profile:
                 self.import_by_device()
         if self.profile:
-            action = f"Delete profile: {self.profile.get_id()}"
+            action = (
+                "Delete profile: "
+                f"{self.profile.get_id()}/{self.profile.get_uuid()}"
```

Comment:
I think it would be nice to specify the id and the uuid with something like `id: .../uuid: ...`